src/isDeepEqual.ts

```diff
@@ -7,7 +7,7 @@ import { purry } from "./purry";
  * objects all props will be compared recursively. The built-in Date and RegExp
  * are special-cased and will be compared by their values.
  *
- * !IMPORTANT: Maps, Sets and TypedArrays, and symbol properties of objects  are
+ * !IMPORTANT: Sets and TypedArrays, and symbol properties of objects  are
```

Comment:
Phrasing is now a broken
```suggestion
 * !IMPORTANT: Sets, TypedArrays, and symbol properties of objects  are
```